libs/dh/core/shell/src/lib/dh-core-shell.routes.ts

```diff
@@ -63,6 +63,14 @@ export const dhCoreShellRoutes: Routes = [
         },
         canActivate: [MsalGuard, PermissionGuard(['fas'])],
       },
+      {
+        path: getPath<BasePaths>('dev-examples'),
+        loadComponent: () => import('@energinet-datahub/dh/processes'),
```

Comment:
nitpick: shouldn't this be `@energinet-datahub/dh/dev-examples/feature-processes` if it is to follow the existing naming convention?